DS/Assignment8/DifferentSort/SortingMain

```diff
@@ -0,0 +1,45 @@
+//Class to call all sorting functions
+public class SortingMain {
+
+	public static void main(String arg[]) {
+
+		MenuAndPrint menuandprint = new MenuAndPrint();
+
+		int[] array = menuandprint.arrayInput();
+		int length = array.length;
+		int[] sorted = new int[length];
+		int choice = menuandprint.menu();
+		try {
+			while (choice != 4) {
+				switch (choice) {
```

Comment:
the switch case would not be executed if choice is 4. Thus, the system will never be able to exit